shared/templates/rsyslog_logfiles_attributes_modify/ansible.template

```diff
@@ -12,14 +12,17 @@
 #   (store the result into array for the case there's shell glob used as value of IncludeConfig)
 - name: '{{{ rule_title }}} - Get IncludeConfig directive'
   ansible.builtin.shell: |
-    set -o pipefail
+   {{% if not 'debian' in product %}}set -o pipefail{{% endif -%}}
     grep -e '$IncludeConfig' {{ rsyslog_etc_config }} | cut -d ' ' -f 2 || true
   register: rsyslog_old_inc
   changed_when: False
 
+# note: on debian operating systems, /bin/sh is a symlink that point to /bin/dash shell. Dash lacks the pipefail option, so the
+#   set -o pipefail
+# line should be escaped.
 - name: '{{{ rule_title }}} - Get include files directives'
   ansible.builtin.shell: |
-    set -o pipefail
+    {{% if not 'debian' in product %}}set -o pipefail{{% endif -%}}
```

Comment:
There is one more problem, and that is a  serious one and it causes the fail of the testing farm CI jobs. On other products, this reduces the whitespace and accidentally removes the newline. For example, when you build `rhel9`, you will see this in the built cotnent:

```
     set -o pipefailgrep -e '$IncludeConfig' {{ rsyslog_etc_config }} | cut -d ' ' -f 2 || true
```

Please make sure this won't happen. Dtto for other occurrences.

Reply:
It should be fixed by now.
Thank you for your patience.